### PR TITLE
UIBULKED-442 Bulk edit form enhancements - part 2

### PR DESCRIPTION
**Pull request by vashjs**
As part of this PR, several main problems were resolved: 

- Dynamically adding headers depending on the state of the form
- Added dividers to the full height of the form, emulating a table
- Added indicators to headers (required/not required)

Refs: [UIBULKED-442](https://folio-org.atlassian.net/browse/UIBULKED-442)

UIBULKED-442

### CODE REVIEW

File: src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {
   Layout,
 } from '@folio/stripes/components';
 
+import { useState } from 'react';
```
Comment (BogdanDenis) [inline src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js:11]: Please move third party imports to the top of the file

File: src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInAppTitle/BulkEditInAppTitle.js [hunk 2]
```diff
@@ -1,31 +1,78 @@
...
+import PropTypes from 'prop-types';
 
 import {
   Col,
   Row,
+  Label,
 } from '@folio/stripes/components';
 
 import css from './BulkEditInAppTitle.css';
+import { FINAL_ACTIONS } from '../../../../../constants';
```
Comment (BogdanDenis) [inline src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInAppTitle/BulkEditInAppTitle.js:11]: Please move this import to be before css

File: src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js [hunk 3]
```diff
@@ -38,15 +40,30 @@ export const BulkEditInApp = ({
...
+  const defaultOptionValue = options[0].value;
+
+  const fieldTemplate = {
+    options,
+    option: defaultOptionValue,
+    actionsDetails: getDefaultActions({
+      option: defaultOptionValue,
+      capability: currentRecordType,
+      options,
+      formatMessage
```
Comment (BogdanDenis) [inline src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js:62]: Do we need to pass `formatMessage` from `intl` here?

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js (hunk 1 above)
Comment (vashjs) [reply src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js]: Fixed

File: src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInAppTitle/BulkEditInAppTitle.js (hunk 2 above)
Comment (vashjs) [reply src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInAppTitle/BulkEditInAppTitle.js:10]: Fixed

File: src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js (hunk 3 above)
Comment (vashjs) [reply src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js:62]: Yes, there is some logic related to generationg actions, and it includes parts which have to be translated

**Review by BogdanDenis — APPROVED** [review]

**Review by UladzislauKutarkin — APPROVED** [review]

**Pull request merged**